<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Atlas</class>
 <widget class="QMainWindow" name="Atlas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atlas editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="toolTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QStackedWidget" name="screen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>FPS:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="fps_counter">
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="position_x">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="position_y">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="position_z">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="view_free">
          <property name="text">
           <string>Free</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="view_editor">
          <property name="text">
           <string>Editor</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="view_player">
          <property name="text">
           <string>Player</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1332</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionRecent_Projects"/>
    <addaction name="actionSave_All"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionWireframe"/>
    <addaction name="actionShadow"/>
    <addaction name="actionLight"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionQbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSun"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionShow_Fullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
    <addaction name="actionAssets"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionLoad_Object">
   <property name="text">
    <string>Load Object</string>
   </property>
  </action>
  <action name="actionPlayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Player</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
  <action name="actionEracer">
   <property name="text">
    <string>Eracer</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="text">
    <string>Brush</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionTextures">
   <property name="text">
    <string>Textures</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionFill_Ground">
   <property name="text">
    <string>Fill Ground</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionProcedural">
   <property name="text">
    <string>Procedural</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew_Mesh">
   <property name="text">
    <string>New Mesh</string>
   </property>
  </action>
  <action name="actionRecent_Projects">
   <property name="text">
    <string>Recent Projects</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="actionShadow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shadow</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionQbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionSun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sun</string>
   </property>
  </action>
  <action name="actionShow_Fullscreen">
   <property name="text">
    <string>Show Fullscreen</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAssets">
   <property name="text">
    <string>Assets</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
